libraries/cli/include/cli/devnet_config.hpp

```diff
@@ -214,7 +214,7 @@ constexpr std::string_view devnet_json = R"foo({
         },
         "execution_options": {
           "disable_gas_fee": false,
-          "disable_nonce_check": false
+          "disable_nonce_check": true
```

Comment:
it should be changed for testnet only

Reply:
why ? Devnet has the same issue and it wont work because of this so we should probably find the fix locally and let the networks run ?